src/qt/test/rpcnestedtests.cpp

```diff
@@ -34,9 +34,6 @@ void RPCNestedTests::rpcNestedTests()
     tableRPC.appendCommand("rpcNestedTest", &vRPCCommands[0]);
     //mempool.setSanityCheck(1.0);
 
-    ECC_Stop(); // Already started by the common test setup, so stop it to avoid interference
```

Comment:
In commit "qt: test: Create at most one testing setup" (fac5bb8f2e135090cc21bdde80ff2adc4c1a1401)

It seems better to keep these lines here than to move them to main with the comment "started by the AppTests later on". Why move code that is specific to apptests outside of apptests?

Similarly it seems like the BasicTestingSetup object in main could just be an apptests member if it's going to be destroyed as soon as apptests finish.

Reply:
Fixed